src/pages/Projects.jsx

```diff
@@ -10,147 +10,155 @@ import Grid from '@mui/material/Grid';
 import Link from '@mui/material/Link';
 import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
+import { useMediaQuery } from '@mui/material';
+// React Icons Import
+import { IconContext } from 'react-icons';
 // Custom Imports
 import Hero from '../components/global/Hero/Hero';
 import projectsList from '../components/projects/projectsList';
 
-const renderingLinks = (href, icon) => {
-  return (
-    <Link
-      component={ReactRouterLink}
-      key={href}
-      to={href}
-      target="_blank"
-      rel="noopener noreferrer"
-      sx={linkStyle}
-    >
-      {icon}
-    </Link>
-  );
+const linkStyle = {
+  color: '#000',
+  textDecoration: 'none',
+  display: 'flex',
+  '&:hover': {
+    color: 'primary.main'
+  }
 };
 
 const boxStyle = (index) => ({
-  mb: { xs: 5, sm: 15 },
-  py: 5,
+  mb: { xs: 5, sm: 10 },
+  py: 2,
   px: 2,
   borderRadius: '25px',
   background: {
     md:
       index % 2 === 0
         ? 'linear-gradient(270deg, rgba(217, 217, 217, 0) 38.54%, rgba(217, 217, 217, 0.4) 82.29%)'
```

Comment:
I think a few people wanted to see the gradients extended a little maybe something like `'linear-gradient(270deg, rgba(217, 217, 217, 0) 25.54%, rgba(217, 217, 217, 0.4) 54.29%)'` I am ok with either or, do whatever you feel is best for this.

Reply:
oh I thought it was solely on the mobile view... I can change for desktop too.

Reply:
I couldn't remember, if it was just mobile ignore my comment.